PowerEditor/src/NppDarkMode.cpp

```diff
@@ -634,10 +725,28 @@ namespace NppDarkMode
 
 		if (IsColorSchemeChangeMessage(lParam))
```

Comment:
change to `if (IsColorSchemeChangeMessage(lParam) || isFromBtn)`
and `void handleSettingChange(HWND hwnd, LPARAM lParam)` to `void handleSettingChange(HWND hwnd, LPARAM lParam, bool isFromBtn)`

also in header to `void handleSettingChange(HWND hwnd, LPARAM lParam, bool isFromBtn = false)`